Scripts/Modules/HierarchyModule.cs

```diff
@@ -60,141 +79,143 @@ List<string> GetFilterList()
 
 		List<HierarchyData> GetHierarchyData()
 		{
-			if (m_HierarchyData == null)
-				return new List<HierarchyData>();
-
-			return m_HierarchyData.children;
+			return m_HierarchyData ?? new List<HierarchyData>();
```

Comment:
Any reason to not assign m_HierarchyData to a new List<HierarchyData>() instead of just returning it?